tests/regr_test.py

```diff
@@ -254,6 +254,12 @@ def main() -> ReturnCode:
 
     code = 0
     for testcase_dir in testcase_directories:
+        assert isinstance(testcase_dir, PackageInfo)
+        if not testcase_dir.is_stdlib:
+            metadata = read_metadata(testcase_dir.name)
+            if metadata.requires_python and not metadata.requires_python.contains(PYTHON_VERSION):
```

Comment:
Same comment here as in `mypy_test.py`: I think you should be looking at the `versions_to_test` local variable here rather than `PYTHON_VERSION`.